code/platformio.ini

```diff
@@ -132,7 +132,7 @@ lib_deps =
     https://github.com/sparkfun/SparkFun_VEML6075_Arduino_Library#V_1.0.3
     https://github.com/pololu/vl53l1x-arduino#1.0.1
     https://github.com/mcleng/MAX6675-Library#2.0.1
-    https://github.com/ElderJoy/esp8266-oled-ssd1306#4.0.1
+    https://github.com/ThingPulse/esp8266-oled-ssd1306
```

Comment:
pin https://github.com/ThingPulse/esp8266-oled-ssd1306/commit/3398c975a6fb058b3a75c704d9f6ebf1312a97f0 or 4.1.0?

Reply:
4.1.0 - tag in my fork of this lib. It is not valid in the original repo.
I tested original library on it's HEAD commit and it works fine for me. So, let's pin it to latest commit #3398c97.